src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -588,7 +590,7 @@ public VideoToAudioResponse VideoAudio(@RequestParam("asset") MultipartFile asse
         if (inputJson.getString("checksum").equals(inputChecksumString)) {
             new File(inputJson.getString("destPath") + outputJson.getString("name") + "/").mkdirs();
             CriteriaAudio criteria = new CriteriaAudio();
-            criteria.setSrcPath(inputJson.getString("destPath")+ fileName);
+            criteria.setSrcPath(inputJson.getString("destPath") + fileName);
 
```

Comment:
this line of code must complete 120 characters

Reply:
that was resolved.